### PR TITLE
Opacity Transition for Dark Mode Toggle Button 

### LINKED ISSUE #452: dark mode toggle button delayed loading

### LINKED ISSUE #513: Dark mode toggle button causes layout shift in navbar
### What happened?

The content of the dark mode toggle button appears to load lazily, causing the surrounding elements to be affected.

image

### PR DESCRIPTION
**Pull request by k4rni**
Fixed delay in discord dark mode button to make it more fluid.

Before:
ezgif-3-652e1f0f1e

After:
ezgif-4-012ec8614b

### CODE REVIEW

**Comment by jaasonw** [comment]
hm, the thing about this one is it does look more fluid but it doesn't actually fix the delay

**Comment by jaasonw** [comment]
https://user-images.githubusercontent.com/8981287/168950987-716d6b7d-ac57-464a-b9d7-4216279f64b9.mp4

**Comment by jaasonw** [comment]
[6:52 PM] EthanThatOneKid: The delay is expected cuz we want it to appear for js users only btw https://github.com/EthanThatOneKid/acmcsuf.com/pull/474#issuecomment-1129483291
[8:11 PM] EthanThatOneKid: i just figured fade-in is better than layout shift
[8:20 PM] jasonw: interesting

Steps to reproduce: Disable javascript
I recoreded this on snapchat bcus i dont have a screen recorder software rn

https://user-images.githubusercontent.com/8981287/168951140-abd1f824-89ea-4514-a40f-06299a795356.mp4

**Comment by EthanThatOneKid** [comment]
> Steps to reproduce: Disable javascript I recoreded this on snapchat bcus i dont have a screen recorder software rn
> 
>  Snapchat-1899418180.mp4

To reproduce the flicker seen in the video, disable cache in addition to disabling JavaScript.

**Comment by jaasonw** [comment]
image

**Review by charliettaylor — APPROVED** [review]
LGTM!

**Review by jaasonw — APPROVED** [review]
I think this is gucci gang as a standalone change but doesn't actually fix #452 so I'm gonna unlink it from the issue

File: src/lib/components/sections/navbar.svelte [hunk 1]
```diff
@@ -53,23 +53,28 @@
...
+      <div class="toggle-container" class:hidden={!jsEnabled}>
+        <!-- true is dark -->
+        <AcmToggle
+          checked={$theme === AcmTheme.Dark}
+          on:toggle={(event) => ($theme = event.detail ? AcmTheme.Dark : AcmTheme.Light)}
+        >
+          <noscript>
+            <style>
+              .dark-toggle {
+                opacity: ;
```
Comment (EthanThatOneKid) [inline src/lib/components/sections/navbar.svelte]: Opacity is set to what?

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]

File: src/lib/components/sections/navbar.svelte (hunk 1 above)
Comment (k4rni) [reply src/lib/components/sections/navbar.svelte]: it's supposed to be 0

Comment (jaasonw) [reply src/lib/components/sections/navbar.svelte]: wait, WTF LMFISDFLKSDFKSDLF HOW DID THAT HAPPEN

Comment (jaasonw) [reply src/lib/components/sections/navbar.svelte]: ok i fix

**Review by EthanThatOneKid — APPROVED** [review]

**Pull request merged**